readme.md

```diff
@@ -366,6 +367,38 @@ add_action('openid-connect-generic-redirect-user-back', function( $redirect_url,
 }, 10, 2); 
 ```
 
+
+#### `openid-connect-generic-register-login-form`
+
+Allow user to add the login form to various pages, such as WooCommerce's checkout page.  It will fire
+whenever the plugin is loaded and pass the login form to the callback.
+
+Provides 1 argument: the login form instance.
+
+```
+add_action ('openid-connect-generic-register-login-form',
+    function ( $login_form ) {
+
+	// show login form at the shopping cart (if not logged in)
+	add_action( 'woocommerce_before_checkout_billing_form',
+                    function () use ( $login_form ) {
+                             $user = wp_get_current_user ();
+                             if (0 == $user->ID) {
+		                // ID 0 is used to indicate user is not logged in.
+                                // Re-use filter logic to generate login page
+               	                print ( $login_form->handle_login_page ('') );
+	                     }
+                     });
+
+        // Add action to set cookie to redirect back to current
+        // (checkout) page after OIDC provided the data
+        add_action( 'woocommerce_before_checkout_billing_form',
+                    array( $login_form, 'handle_redirect_cookie' ) );
+
+    }
+);
```

Comment:
This example leads me to believe that there is an attempt to get the login form which should already be available by using the `openid_connect_generic_login_button` shortcode.